packages/now-next/src/index.ts

```diff
@@ -489,7 +489,31 @@ export const build = async ({
       ...staticFiles,
       ...staticDirectoryFiles,
     },
-    routes,
+    routes: [
+      // Next.js page lambdas
+      ...Object.keys(lambdas).map(lambdaName => {
+        const lambdaPath = path.join('/', lambdaName);
+
+        // All page lambda routes are normalized except for `/index`
+        const isRoot = lambdaPath === path.join('/', entryDirectory, '/index');
```

Comment:
```suggestion
        const isRoot = lambdaPath === path.join('/', entryDirectory, 'index');
```
I should have suggested we consistently do this